docs/topics/resources/vert-x/vertx-fmp-deployment.yaml

```diff
@@ -0,0 +1,15 @@
+spec:
+  template:
+    spec:
+      containers:
+        - name: vertx
+          env:
+            - name: KUBERNETES_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: metadata.namespace
+            - name: JAVA_OPTIONS
+              value: '-Dvertx.cacheDirBase=/tmp -Dvertx.jgroups.config=default-configs/default-jgroups-kubernetes.xml -Djava.net.preferIPv4Stack=true'
+            - name: JAVA_ARGS
+              value: '-Dvertx.cacheDirBase=/tmp -Dvertx.jgroups.config=default-configs/default-jgroups-kubernetes.xml -Djava.net.preferIPv4Stack=true -cluster'
```

Comment:
@cescoffier do we really need to duplicate this? From what I understood from [FMP documentation]( https://maven.fabric8.io/#generator-vertx), more specifically from here

> You can pass application parameter by setting the `JAVA_ARGS` env variable. You can pass system properties either using the same variable or using `JAVA_OPTIONS`. For instance, create src/main/fabric8/deployment.yml with the following content to configure `JAVA_ARGS`:

It looks like it should be enough to specify the java args and system properties in just `JAVA_ARGS`

Reply:
@spisiakm @cescoffier I would like to know the answer to this as well. I noted that both were in the example.

Reply:
Fixed!

Reply:
I don't quite understand, does that mean that both of those declarations should be there?

Reply:
Both arguments were in the code sample provided by @cescoffier. Until I hear otherwise, I am leaving them both.

Reply:
@huntley, please seek confirmation from @cescoffier to make sure this is indeed correct. We need an authoritative answer from either engineering or QE before merging. Thanks.

Reply:
Sorry, just had a look.

It should be:
```
            - name: JAVA_OPTIONS
              value: '-Dvertx.cacheDirBase=/tmp -Dvertx.jgroups.config=default-configs/default-jgroups-kubernetes.xml -Djava.net.preferIPv4Stack=true'
            - name: JAVA_ARGS
              value: '-cluster'
```

Reply:
@cescoffier Thank you, much appreciated.